kubemarine/kubernetes/__init__.py

```diff
@@ -831,7 +838,7 @@ def upgrade_workers(upgrade_group: NodeGroup, cluster: KubernetesCluster, **drai
         fix_flag_kubelet(node)
 
         node.sudo(
-            "kubeadm upgrade node --certificate-renewal=true --patches=/etc/kubernetes/patches && "
+            "kubeadm upgrade node {flags} && "
```

Comment:
Linter fails CI with warning that `flags` is unused. It seems we should use python f-string here instead of simple string
```suggestion
            f"kubeadm upgrade node {flags} && "
```